app/src/gui/components/notebook/index.tsx

```diff
@@ -331,6 +341,16 @@ export default function NotebookComponent(props: NotebookComponentProps) {
                   chips={false}
                 />
               </Typography>
+              {template_id && (
+                <Typography
+                  variant="body1"
+                  gutterBottom
+                  sx={{marginBottom: '16px'}}
+                >
+                  <strong>Template Used: </strong>
+                  <span>{template_id}</span>
+                </Typography>
+              )}
```

Comment:
Oh I see - so the MetadataRenderer component actually fetched the metadata property from the DB - which is why you needed to elevate that behaviour into this component to make it conditional - bit of a pain but works I guess.

Reply:
Looking forward to metadata refactor